lib/passageidentity/user_api.rb

```diff
@@ -215,23 +159,24 @@ def list_devices(user_id:)
         response.devices
       rescue Faraday::Error => e
         raise PassageError.new(
-          'failed to delete Passage User Device',
           status_code: e.response[:status],
           body: e.response[:body]
         )
       end
     end
 
     def signout(user_id:)
```

Comment:
same here, it needs to return true while `revoke_refresh_tokens` does not

Reply:
Same as above. Does the api return true?